includes/multi-currency/Compatibility/WooCommerceNameYourPrice.php

```diff
@@ -63,7 +63,7 @@ public function add_initial_currency( $cart_item, $product_id, $variation_id ) {
 
 		$nyp_id = $variation_id ? $variation_id : $product_id;
 
-		if ( \WC_Name_Your_Price_Helpers::is_nyp( $nyp_id ) && isset( $cart_item['nyp'] ) ) {
+		if ( class_exists( '\WC_Name_Your_Price_Helpers' ) && \WC_Name_Your_Price_Helpers::is_nyp( $nyp_id ) && isset( $cart_item['nyp'] ) ) {
```

Comment:
This check makes sense since we were assuming the Name Your Price plugin would always have the helper class when it may not.